### PR TITLE
replaced helmet with next/head and fixes titles, and new draft screeen

### PR DESCRIPTION
**Pull request by davodesign84**

### CODE REVIEW

File: packages/kauri-web/index.js [hunk 1]
```diff
@@ -3,7 +3,6 @@ require("@babel/register")({
 });
 const express = require("express");
 const compression = require("compression");
-const helmet = require("helmet");
```
Comment (rej156) [inline packages/kauri-web/index.js]: Add back, not related, this is a express helmet module :P

File: package.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (rej156) [inline package.json]: Wrong file location :P

File: packages/kauri-web/lib/hljs.tsx [hunk 3]
```diff
@@ -55,13 +55,13 @@ hljs.registerLanguage(
   require("highlight.js/lib/languages/typescript")
 );
 
-export default ({ children }) => (
-  <BareHighlight
+export default ({ children }: any) => (
+  <Highlight
```
Comment (rej156) [inline packages/kauri-web/lib/hljs.tsx]: What was the agenda here?

File: packages/kauri-web/index.js (hunk 1 above)
Comment (davodesign84) [reply packages/kauri-web/index.js]: Ohh right that's the headers module, cool will do.

File: packages/kauri-web/lib/hljs.tsx (hunk 3 above)
Comment (davodesign84) [reply packages/kauri-web/lib/hljs.tsx]: Cause we had a couple of issues with languages not supported, and we want to expand eventually so might as well do it :P

File: package.json (hunk 2 above)
Comment (davodesign84) [reply package.json]: oops

File: packages/kauri-web/lib/hljs.tsx (hunk 3 above)
Comment (rej156) [reply packages/kauri-web/lib/hljs.tsx]: Err they're manually context replaced via next.config.js + required in all those lines above where you edited here

Won't work until you remove those lines
Feel free to fuck with webpack, heard you like

Comment (rej156) [reply packages/kauri-web/lib/hljs.tsx]: 😢 webpack

Comment (davodesign84) [reply packages/kauri-web/lib/hljs.tsx]: daaaaamn but it seems to work when running locally

**Pull request merged**